build/build.xml

```diff
@@ -1,332 +0,0 @@
-<?xml version="1.0" ?>
```

Comment:
Don't we need build script? I feel anyone can build project using `phing` like `make` projects. Otherwise if we define build process on Github action. It may be hard to build in case local build. Have to follow all build steps manually.